test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1381,6 +1381,8 @@ BOOST_AUTO_TEST_CASE(internal_constructor)
 			constructor() {}
 		}
 	)";
+	// Disabled via yul because it will throw an error instead of
+	// returning empty bytecode.
```

Comment:
Where was it disabled?

Reply:
It was not enabled. All other tests here can be enabled.

Reply:
ah ok

Reply:
This comment vanished now, was that intended?

Reply:
It was part of another PR that got merged in the meantime.